es-app/src/CollectionSystemManager.cpp

```diff
@@ -98,40 +103,14 @@ void CollectionSystemManager::deinit()
 	}
 }
 
-void CollectionSystemManager::saveCustomCollection(SystemData* sys)
```

Comment:
This is an example of a method that I think belongs to the CollectionSystem-subclass to be implemented. See down below.

Reply:
Agreed.